.travis.yml

```diff
@@ -125,7 +126,6 @@ jobs:
         - pip install --user ansible
       script:
         - make test-e2e-ansible
-        - make test-e2e-ansible-molecule
 
     # Test subcommands
     - <<: *test
```

Comment:
Shows that we can remove all:  `/operator-sdk/internal/scaffold/`
Required check if has some part of code which will require it.

Reply:
Some of it is still used for scaffolding customize templates (eg: in `generate/kustomize/manifests.go`) and in `add crd`.
With the removal of `add crd`, Ill clean up internal/scaffold too.